dask_sql/context.py

```diff
@@ -481,7 +487,7 @@ def sql(
                 for df_name, df in dataframes.items():
                     self.create_table(df_name, df, gpu=gpu)
 
-            rel, select_fields, _ = self._get_ral(sql)
+            rel, select_fields, _ = self._get_ral(sql, config_options)
```

Comment:
We can avoid passing this explicitly and use the `dask_config.get` directly within `_get_ral`. 

It has the added advantage of being able to use config options set in a dask yaml or explicitly set in the context that's not explicitly passed in here